packages/graphql/tests/e2e/subscriptions/auth/roles.int.test.ts

```diff
@@ -18,7 +18,7 @@
  */
 
 import { Driver } from "neo4j-driver";
-import supertest, { Response } from "supertest";
+import { Response } from "supertest";
```

Comment:
```suggestion
import type { Response } from "supertest";
```

Can we specify it as a type import?